### PR TITLE
Update default Envoy image version and OS

### PR DESCRIPTION
**Pull request by ishustava**
Update Envoy image to alpine version `1.14.2` for all gateways.

### CODE REVIEW

**Review by kschoche — APPROVED** [review]
Looks good to me! 🥇

**Review by lkysow — APPROVED** [review]

**Comment by ishustava** [comment]
I'll merge this after the terminating gateways PR (#503) is merged.

**Pull request merged**
